docs/private-node-attributes.md

```diff
@@ -0,0 +1,26 @@
+# Private attributes of the NeoFS storage nodes
+
+Storage nodes declare key-value string attributes when applying to enter the
+network map. In general, any attributes can be declared, however, some of them
+may be subject to restrictions. In particular, to specify so-called private
+attributes, the node must be in the corresponding access list.
+
+## Access lists
+
+These lists are stored in the NeoFS NNS. For each private attribute value
+`KEY=VALUE`, there is a domain with name `MD5(VALUE).MD5(KEY).private-node-attributes.neofs`
+and records of `TXT` type with HEX-encoded public keys. Here `MD5` is a
+HEX-encoded MD5 checksum. If the domain exists and has non-empty list of public
+key records, only storage nodes with keys from this list will be able to set
+this attribute value. The Inner Ring will deny everyone else access to the
+network map.
+
+Note that if domain for the attribute exists but has no records, attribute is
+non-private.
+
+### Domain record format
+
+For each public key, a record is created - a structure with at least 3 fields:
+1. `ByteString` with name of the corresponding domain
+2. `Integer` that should be `16` (TXT records)
+3. `ByteString` with HEX-encoded public key
```

Comment:
For any address a key can be derived. Any address can be stored using https://github.com/neo-project/proposals/pull/133. Maybe this would be a bit better?

Reply:
sounds like a working option

> Maybe this would be a bit better?

not ready to answer right now. Can we outline particular problem/ability such approach may solve/provide? so that the transition is justified not only by technical feasibility